recipes/ride-mode

```diff
@@ -0,0 +1 @@
+(twtxt :fetcher github :repo "deadblackclover/ride-mode")
```

Comment:
```suggestion
(ride-mode :fetcher github :repo "deadblackclover/ride-mode")
```

Reply:
Sorry, I was copying from my other package. Thx for fixing it!